test/PspHgh.jl

```diff
@@ -91,18 +96,24 @@ end
     @test (res[1] < res[2]) == (res[3] < res[2])
 
     for i in 1:2, l in 0:2
-        qcut = qcut_psp_projector(psp, i, l)
-        res = eval_psp_projector_fourier.(psp, i, l, [qcut - ε, qcut, qcut + ε])
-        @test (res[1] < res[2]) == (res[3] < res[2])
+        qcut1 = qcut_psp_projector(psp, i, l)
```

Comment:
Why the rename ?

Reply:
Ah now I get why these things are needed. Here I would actually just put the code block above inside a `let ... end` then the `qcut` is not on the global scope.